cmd/crc/cmd/start.go

```diff
@@ -38,6 +38,7 @@ var startCmd = &cobra.Command{
 
 var (
 	startCmdFlagSet = initStartCmdFlagSet()
+	stopChronyd     = "stop-chronyd"
```

Comment:
For what it's worth, I would have gone with an environment variable (CRC_DEBUG_ENABLE_TIME_SKEW with a value in seconds) rather than a command line flag.

Reply:
> CRC_DEBUG_ENABLE_TIME_SKEW with a value in seconds) rather than a command line flag.

@cfergeau  This is just a bool flag and yes we can go with env variable if you prefer.

Reply:
It should explicitly set the time in the VM, or you are going to have a short window when `chronyd` is running, and you are going to race with this and assume the time is not going to be reset through NTP before you get a chance to disable NTP.

But I'm also wondering if it would not be better to just run externally a loop trying to ssh into the VM and shutdown NTP, this way no need for very special code in crc, and all time hacks would be located in the same place (integration tests)

Reply:
@cfergeau it's not with integration test, if test failed on the CI I want to do it locally then this gives me a better way to recreate env then running integraiton test locally. About the race condition it the time sync with ntp on the VM then also host time differ and we will have failed test.

Reply:
> it's not with integration test, if test failed on the CI I want to do it locally then this gives me a better way to recreate env then running integraiton test locally.

You need to know and run some manual steps locally already if you want to reproduce the CI environment locally, so this does not feel like a very strong argument :-/

> About the race condition it the time sync with ntp on the VM then also host time differ and we will have failed test.

But we don't want CI to fail because it triggered a race condition in the integration test implementation, do we?